ci-operator/templates/openshift/installer/cluster-launch-installer-custom-test-image.yaml

```diff
@@ -632,6 +653,41 @@ objects:
         TF_LOG=debug openshift-install --dir=/tmp/artifacts/installer create cluster 2>&1 | grep --line-buffered -v password &
         wait "$!"
 
+        if [[ "${CLUSTER_TYPE}" == "gcp" ]] && [[ "${CLUSTER_ENABLE_NESTED_VIRT-false}" == "true" ]]; then
+          ### enable nested-virt ###
+          export KUBECONFIG=/tmp/artifacts/installer/auth/kubeconfig
+          oc get machineset -n openshift-machine-api -o json >/tmp/machinesets.json
+          MACHINE_IMAGE=$(jq -r .items[0].spec.template.spec.providerSpec.value.disks[0].image /tmp/machinesets.json)
+          sed -i 's/'"$MACHINE_IMAGE"'/'"$CLUSTER_NESTED_VIRT_IMAGE"'/g' /tmp/machinesets.json
+          oc apply -f /tmp/machinesets.json
```

Comment:
ok, this is what's needed here (L657- `oc scale --replicas=0...`:
```
### enable nested-virt ###
export KUBECONFIG=/tmp/artifacts/installer/auth/kubeconfig
oc get machineset -n openshift-machine-api -o json >/tmp/machinesets.json
# this image lives in gce-devel project: "rhcos43-nested-virt"
jq '.items[0].spec.template.spec.providerSpec.value.disks[0].image = "rhcos43-nested-virt"' /tmp/machinesets.json > /tmp/machinesets-nested.json
oc apply -f /tmp/machinesets-nested.json
```
then continue...to L663